index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    <body>
+        <div class="all">
+            <div class="top-container">
+                <div class="header">
+                    <div class="logo"><p class="logo">My Logo</p></div>
+                    <div class="menu"><img src="image/Enabled.png" alt="menu"/></div>
+                </div>
+                <div class="main">
+                    <div class="bg">
+                    </div>
+                    <div class="name">
+                        <p>I’m Eric Tuyishimire Glad to see you!</p>
+                    </div>
+                    <div class="content">
+                        <p>
+                            I’m a software developer! I can help you build a product , 
+                            feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, 
+                            don’t hestiate to contact me.</p>
+                    </div>
+                    <div class="connect">
+                        <p>LET'S CONNECT</p>
+                    </div>
+                    <div class="social">
+                        <img src="image/social.png" alt="menu"/>
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon should have a link (`a` tag)

Reply:
- [ ] Reduce space here to make it more similar to design 
![image](https://user-images.githubusercontent.com/62705947/171380442-c8fdd921-25df-4410-b70f-907052853e85.png)